examples/plotting/trefoil.py

```diff
@@ -1,3 +1,13 @@
+''' This example plots a Radiation Warning Symbol (Trefoil). It demonstrates
```

Comment:
```suggestion
''' This example shows a Radiation Warning Symbol (Trefoil). It demonstrates
```
agree w/ @mattpap "shows" is better. I do think "a" is better then "the" in this case, since the plot shows one of them and not, say two or three of them. But I am fine with either.